index.css

```diff
@@ -321,318 +234,153 @@ a {
   margin: 12px 0;
 }
 
-.work-section1 > .frame-1 {
+.project > .inline-summary {
   display: flex;
   flex-direction: row;
   align-items: center;
   padding: 0;
   position: static;
-  width: 295px;
+  width: 236px;
   height: 32px;
   left: 0;
-  top: 92px;
+  top: 48px;
+  justify-content: space-between;
   flex: none;
+  list-style-type: none;
   flex-grow: 0;
   margin: 12px 0;
 }
 
-.work-section1 > .instruction-1 {
+.project > .content {
+  position: static;
   width: 295px;
   height: 76px;
   left: 0;
-  top: 136px;
+  top: 92px;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 400;
-  font-size: 18px;
+  font-size: 17px;
   line-height: 24px;
-  color: #344563;
+  flex: none;
 }
 
-.work-section1 > .categorie1-work > ul {
+.project > .categorie-work {
   display: flex;
   flex-direction: row;
   align-items: center;
   padding: 0;
   position: static;
-  width: 200px;
+  width: 203px;
   height: 24px;
   left: 0;
-  top: 224px;
+  top: 180px;
+  color: #6070ff;
   justify-content: space-between;
+
+  /* Inside auto layout */
   list-style-type: none;
   flex: none;
-  order: 2;
   flex-grow: 0;
-  margin: 12px  90px -30px 0;
+  margin: 12px 0;
 }
 
-.work-section1 > .see-button1 {
+.button {
   display: flex;
   flex-direction: row;
   align-items: center;
-  padding: 12px;
-  width: 130px;
-  height: 40px;
-  left: 0;
-  top: calc(50% - 48px / 2);
-  border: 1px solid #6070ff;
-  box-sizing: border-box;
-  border-radius: 8px;
+  padding: 4px 12px;
+  justify-content: center;
   position: static;
-  background-color: #fff;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 17px;
-  line-height: 24px;
-  text-align: center;
-  letter-spacing: 0.03em;
-  color: #396df2;
-  margin: 50px 170px -40px 0;
-}
+  width: 65px;
+  height: 26px;
+  left: 12px;
+  top: 4px;
+  color: #6070ff;
 
-.work-section2 {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 16px;
-  position: static;
-  width: 327px;
-  height: 552px;
-  left: calc(50% - 327px / 2);
-  top: 114px;
-  background: #fff;
-  border: 1px solid #dfe1e6;
-  box-sizing: border-box;
-  border-radius: 16px;
+  /* Primary/B50 */
+  list-style-type: none;
+  background: #ebebff;
+  border-radius: 8px;
+  border-style: none;
+  align-content: center;
   flex: none;
-  order: 0;
+  order: 2;
   flex-grow: 0;
-  margin: 88px 0;
+  margin: 0 8px;
 }
 
-.work-section2 > .tonic {
+.see-button {
   position: static;
-  width: 295px;
-  height: 36px;
-  left: 0;
-  top: 0;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
-  color: #172b4d;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  margin: 12px 0;
-}
 
-.work-section2 > .frame {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 0;
-  position: static;
-  width: 285px;
-  height: 32px;
+  /* width: 195px;
+  height: 72px; */
```

Comment:
- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.